src/containers/Etherspot.tsx

```diff
@@ -65,35 +65,49 @@ const Etherspot = ({
   smartWalletOnly,
   hideWalletBlock = false,
   onLogout,
-}: EtherspotProps) => (
-  <ThemeProvider theme={merge({}, darkTheme, themeOverride)}>
-    <EtherspotContextProvider
-      provider={provider}
-      chainId={chainId}
-      etherspotSessionStorage={etherspotSessionStorage}
-      onLogout={onLogout}
-      smartWalletOnly={smartWalletOnly}
-    >
-      <style>
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Regular.css');
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Medium.css');
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Bold.css');
-      </style>
-      <ComponentWrapper>
-        <TransactionBuilderModalContextProvider>
-          <TransactionsDispatcherContextProvider>
-            <TransactionBuilderContextProvider
-              defaultTransactionBlocks={defaultTransactionBlocks}
-              hiddenTransactionBlockTypes={hiddenTransactionBlockTypes}
-              hideAddTransactionButton={hideAddTransactionButton}
-              showMenuLogout={showMenuLogout}
-              hideWalletBlock={hideWalletBlock}
-            />
-          </TransactionsDispatcherContextProvider>
-        </TransactionBuilderModalContextProvider>
-      </ComponentWrapper>
-    </EtherspotContextProvider>
-  </ThemeProvider>
-);
+}: EtherspotProps) => {
+  const [activeTheme, setActiveTheme] = useState(darkTheme);
+
+  useEffect(() => {
+    const currentTheme = localStorage.getItem("current-theme");
+    if(!currentTheme){
+      localStorage.setItem("current-theme", 'DARK');
+      return;
+    }
+    setActiveTheme(getTheme(currentTheme));
+  }, []);
+
+  return (
+    <ThemeProvider theme={merge({}, activeTheme, themeOverride)}>
+      <EtherspotContextProvider
+        provider={provider}
+        chainId={chainId}
+        etherspotSessionStorage={etherspotSessionStorage}
+        onLogout={onLogout}
+        smartWalletOnly={smartWalletOnly}
+        changeTheme={setActiveTheme}
+      >
+        <style>
+          @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Regular.css'); @import
+          url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Medium.css'); @import
+          url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Bold.css');
```

Comment:
```suggestion
        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Regular.css');
        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Medium.css');
        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Bold.css');
```

Reply:
I think this is done by the formatter for some reason, so its fine to just leave this alone